k8s/business-create-ui.cmap.param

```diff
@@ -10,9 +10,10 @@ TAG_NAME="dev"
 LEGAL_API_URL="https://legal-api-dev.pathfinder.gov.bc.ca/api/v1/businesses/"
 AUTH_API_URL="https://auth-api-dev.pathfinder.gov.bc.ca/api/v1/"
 BUSINESSES_URL="https://dev.bcregistry.ca/cooperatives/auth/"
+AUTH_URL="https://dev.bcregistry.ca/cooperatives/auth/"
```

Comment:
I think an entry for this is needed in the param file too.

Reply:
yes ran into this when I tried to create the prod env. Just uploaded the fix

Reply:
It's now in both param and both json files